core/src/test/resources/HistoryServerExpectations/stage_with_peak_metrics_expectation.json

```diff
@@ -33,6 +33,16 @@
   "shuffleLocalBytesRead" : 0,
   "shuffleReadBytes" : 0,
   "shuffleReadRecords" : 0,
+  "shuffleCorruptMergedBlockChunks" : 160,
+  "shuffleMergedFetchFallbackCount" : 32,
+  "shuffleMergedRemoteBlocksFetched" : 160,
+  "shuffleMergedLocalBlocksFetched" : 240,
+  "shuffleMergedRemoteChunksFetched" : 192,
+  "shuffleMergedLocalChunksFetched" : 160,
+  "shuffleMergedRemoteBytesRead" : 1600,
+  "shuffleMergedLocalBytesRead" : 1600,
+  "shuffleRemoteReqsDuration" : 0,
```

Comment:
if mergedRemoteReqDuration > 0, then shuffleRemoteReqsDuration should be greater than mergedRemoteReqDuration. Can we update this?
The existing values for shuffleLocalBytesRead, and shuffleReadBytes  also don't make sense now because if we are saying that there were merged local blocks, then these value should be > 0. 
We can either change or ignore. I am good with either.